config/config.go

```diff
@@ -129,6 +129,7 @@ type (
 		Log                log.GlobalConfig                `yaml:"log"`
 		SubLogs            map[string]log.GlobalConfig     `yaml:"subLogs"`
 		Genesis            genesis.Genesis                 `yaml:"genesis"`
+		MptrieLogPath      string                          `yaml:"mptrieLogPath"`
```

Comment:
`System` at L103 above is more proper to put this config

Reply:
Done, has been moved into `System`